ops/ansible/roles/bfd-db-migrator/templates/bfd-db-migrator-service.sh.j2

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+export DATABASE_URL='{{ db_migrator_db_url }}'
+export DATABASE_USERNAME='{{ db_migrator_db_username }}'
+export DATABASE_PASSWORD='{{ db_migrator_db_password }}'
+
+LOGS_DIR='{{ db_migrator_dir }}/'
+
+# Either don't set this variable, or set it to one of: BENEFICIARY, CARRIER, DME, HHA, HOSPICE, INPATIENT, OUTPATIENT, PDE, SNF
+# export DATA_SET_TYPE_ALLOWED="BENEFICIARY"
```

Comment:
This looks like carryover from the pipeline script. I don't understand how it would be used in the migrator. Is there a use case for it here?

Reply:
Thanks!